src/icons/Engine.vue

```diff
@@ -0,0 +1,25 @@
+<template>
```

Comment:
Let's add props here as well

```suggestion
<script setup lang="ts">
    import type { IconProps } from './interface';
    const { color = 'white' } = defineProps<IconProps>();
</script>

<template>
```